### PR TITLE
fix readme useRouter "Code Splitting" part

### PR DESCRIPTION
**Pull request by fyangstudio**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

**Comment by yuanyan** [comment]
这里 rax-use-router 里有一个判断逻辑 promise，我理解现在是不需要用户去关注返回的类型的：
https://github.com/alibaba/rax/blob/master/packages/rax-use-router/src/index.js#L163

**Comment by wssgcg1213** [comment]
@fyangstudio https://github.com/alibaba/rax/blob/master/packages/rax-use-router/src/index.js#L166   这一行可以顺便删掉

File: README.md [hunk 1]
```diff
@@ -239,7 +239,11 @@ const config = () => {
         routes: [
           {
             path: '',                 // www.example.com/bar
-            component: () => import(/* webpackChunkName: "bar" */ './Bar'),
+            component: () => import(/* webpackChunkName: "bar" */ './Bar')
+              // interop-require see: https://www.npmjs.com/package/interop-require
+              .then(obj => obj && obj.__esModule ? obj.default : obj)
```
Comment (yuanyan) [inline README.md]: 这里一个 then 应该就足够了吧

Comment (fyangstudio) [reply README.md]: 2 个 then 时，第一个 then 处理模块，后一个专注处理组件 Bar => <Bar />。 一个的话如果写 Bar => <Bar. default /> 时 default 可能不太明确是打包工具的概念还是 Rax 的概念。

**Pull request merged**